adoc/admin-ses-integration.adoc

```diff
@@ -87,36 +91,45 @@ spec:
       readOnly: false
 *EOF*
 ----
-. Verify that the pod exists and its status: 
+//TODO: Where do I get the input for POD_NAME, CONTAINER_NAME and IMAGE_NAME?
+//Can I just input any names I like?
+//Where do i get information for RBD_POOL and the monitors and their ports?
+. Verify that the pod exists and check its status:
 +
 
 ----
 kubectl get pod
 ----
-. Once the pod is running, check the mounted volume: 
+. Once the pod is running, check the mounted volume:
 +
 
 ----
 kubectl exec -it CONTAINER_NAME -- df -k ...
 /dev/rbd1               999320      1284    929224   0% /mnt/rbdvol
 ...
 ----
+//TODO: Could you indicate what this output means?
+//What are the numbers and what do they signify?
 
-
-In case you need to delete the pod, run the following command: 
-
+In case you need to delete the pod, run the following command:
+//TODO: why would I need to delete the pod that I just created?
```

Comment:
Just to make it a round trip operation, when user no longer requires to use pod and wish to remove pod resource. 
I guess this step could be removed if it is not required.